apps-rendering/src/components/blockquote.tsx

```diff
@@ -20,24 +19,44 @@ const styles = (format: ArticleFormat): SerializedStyles => css`
 	font-style: italic;
 	position: relative;
 	margin: ${remSpace[4]} 0 ${remSpace[9]} 0;
-	padding: 0 ${remSpace[6]};
 
-	svg {
-		height: 62.4px;
-		margin-left: -9.2625px;
-		margin-bottom: -12px;
-		margin-top: 4.4px;
+	> svg {
+		width: 18px;
+		height: 12px;
```

Comment:
I think ideally you’d have this be defined relatively to the text size?

```suggestion
		width: 1.5em;
		height: 1em;
```

Reply:
I think regardless of what you decide on the above point it would be useful to have this defined in relative units? So even if not `em`s, then `rem`s?